app/models/messages/MessageData.scala

```diff
@@ -67,6 +67,7 @@ case class MessageData(
         .map(_.map(_.referenceNumber))
         .getOrElse(Seq.empty)
 
+  val conveyanceRefNumberYesNoPresent: Boolean = TransitOperation.isSecurityTypeInSet && Consignment.modeOfTransportAtTheBorder == Option("4")
```

Comment:
`Consignment.modeOfTransportAtTheBorder.contains("4")`

Could also put 4 in a constants file if we have one? Give someone new to the code a bit of context as to what "4" means